ShopperInsights/src/main/java/com/braintreepayments/api/ShoppingInsightsCreateBody.kt

```diff
@@ -0,0 +1,66 @@
+package com.braintreepayments.api
+
+import org.json.JSONArray
+import org.json.JSONObject
+
+internal class ShoppingInsightsCreateBody {
```

Comment:
I think this functionality should all live in the `ShopperInsightsRequest` class

Reply:
fixed!